enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/actions/DownloadWalletAction.java

```diff
@@ -105,7 +110,22 @@ public void actionPerformed(ActionEvent ev) {
                 if (p.getDbUser() != null && p.getDbPassword() != null) {
                     
                     JDBCDriver[] drivers = JDBCDriverManager.getDefault().getDrivers("oracle.jdbc.OracleDriver"); //NOI18N
+                    JDBCDriver jarsPresent = null;
+                    
                     if (drivers.length > 0) {
+                        
+                        // prefer a driver that actually defines some JARs.
+                        for (JDBCDriver d : drivers) {
+                            if (isAvailable(d)) {
+                                jarsPresent = d;
+                                break;
+                            }
+                        }
+                        if (jarsPresent == null) {
+                            jarsPresent = drivers[0];
+                            // PENDING: should be shown to the user ?
+                            LOG.log(Level.WARNING, "Unable to find driver JARs for wallet {0}, using fallback driver: {1}", new Object[] { walletPath, jarsPresent.getName() });
+                        }
```

Comment:
Shouldn't we display message that the driver is not installed here? Driver with no jars is useless here.

Reply:
For NB IDE usage, it's not an issue, as driver's jars can be configured when Connect... action is used. But yes, I can probably display an info dialog - but still create the connection.

Reply:
Addressed in 7ce696e